lib/pr.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import annotations
```

Comment:
P4 nit: rename to `lib/gh/pr.py` ?

I want to move most of what's in spec/gh, eventually, pretty sure.